common-npm-packages/docker-common/containerimageutils.ts

```diff
@@ -304,7 +301,7 @@ export function getBaseImageDigestDockerFile(dockerFileContentPath: string): str
             }
         }
 
-        return null;
+        return baseImageData[0];
```

Comment:
This would end up returning the container image name if no digest is present.  Based on the code comment at the top of this method, isn't the desired behavior to return null if no digest is present?  This would also seem to align with the call from pipelineutils.ts, which falls back to the old logic if no digest is found in the FROM line.